pkg/model/api/v1/dashboard_test.go

```diff
@@ -25,6 +25,7 @@ import (
 )
 
 type TimeSeriesSpec struct {
+	// TODO: show_legend needs to be removed in favor of new spec
```

Comment:
Just for the record, this is just a random struct I created for the test, so it doesn't need to follow the actual definition of TimeSeries panel :)